AppPackages/Settings/Sources/Settings/SettingsView.swift

```diff
@@ -14,16 +14,33 @@ import SwiftUI
 
 public struct SettingsView: View {
 
-	public init() {}
+    @AppStorage("restTimeMinutes") private var restTimeMinutes = 1
+    @AppStorage("restTimeSeconds") private var restTimeSeconds = 5
 
-	public var body: some View {
-		NavigationStack {
-			Text("This is the Settings View")
-				.navigationTitle("Settings")
-		}
-	}
+    public init() {}
+
+    public var body: some View {
+        NavigationStack {
+            HStack {
```

Comment:
Nice work and looks good to me!

Do you mind putting it inside a `List`?

Reply:
That does look much better.
<img width=250px src="https://github.com/user-attachments/assets/28bd7ad7-dc2c-4130-9f12-f6f03867d7e7"/>